frontend/amundsen_application/api/metadata/v0.py

```diff
@@ -525,8 +459,13 @@ def _log_update_table_tags(*, table_key: str, method: str, tag: str) -> None:
         _log_update_table_tags(table_key=table_key, method=method, tag=tag)
 
         metadata_status_code = _update_metadata_tag(table_key=table_key, method=method, tag=tag)
-        search_status_code = _update_search_tag(
-            key=table_key, resource_type=ResourceType.Table, method=method, tag=tag)
+        search_method = method if method == 'DELETE' else 'POST'
```

Comment:
Method for updating now POST because it has the capability to update and create new fields for documents on ES

Reply:
does FE not post to this endpoint as well

Reply:
no it PUTs and I decided not to change it because I would have to update metadata to do the same as well